src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -669,6 +662,7 @@ public Object onTraversalSuccess() {
                 .setGatk4IntegrationParameters(gatk4Parameters)
                 .setSizePerColumnPartition(variantContextBufferSize)
                 .setSegmentSize(segmentSize)
+                .setFailIfUpdating(failIfArrayExists)
```

Comment:
Can you explain why this change is part of this PR, and why there are no new tests to cover this change? It's unrelated to the sample-with-spaces fix. If it was to address https://github.com/broadinstitute/gatk/issues/3815, wasn't that already addressed by patching `overwriteOrCreateWorkspace()` to throw if the workspace already exists, or was that fix incomplete? @kgururaj @lbergelson

Reply:
I guess there is duplication here - GenomicsDB had a flag in the C++ modules to fail if writing to an existing array. The current PR just passes the correct value to the C++ side.

I think Louis wrote a patch to handle this in the GATK code as well.